entity/store.go

```diff
@@ -260,7 +279,16 @@ func (s *store) ReadEntities(entityType string, q query.Query) ([]etre.Entity, e
 	mgoQuery := translateQuery(q)
 
 	var entities []etre.Entity
-	err = c.Find(mgoQuery).All(&entities)
+	if len(f.ReturnLabels) == 0 {
+		err = c.Find(mgoQuery).All(&entities)
+	} else {
+		selectMap := map[string]int{}
+		selectMap["_id"] = 0 // Mongo requires _id to be explicitly excluded
```

Comment:
Oh, I need to swap these two lines so we exclude it by default, then if user includes it they get it, else they don't get it because they didn't include it and we explicitly suppressed it.

Reply:
The for loop below will overwrite this and they'll get `_id`.